quorum/resource_quorum_bootstrap_istanbul_extradata.go

```diff
@@ -30,6 +30,13 @@ func resourceBootstrapIstanbulExtradata() *schema.Resource {
 				Required:    true,
 				ForceNew:    true,
 			},
+			"ibft2_mode": {
```

Comment:
To accommodate QIBFT, maybe we can change this to `mode` with supported values: `ibft1`, `ibft2` and `qibft`.
Although in the nutshell, `ibft2` and `qibft` may use the same logic but for clarity it's good to separate them out.

Reply:
I thought about that. I didn't add as I wasn't sure what is going to be `qibft` and normally don't add something for the future. 

However, I can add `ibft1` and `ibft2` to be clear for now and then later we can add `qibft` when we are sure about the logic.

Reply:
confirmed with @jframe that it won't be the same logic (and is not even completely final).

so for now, only `ibft1` and `ibft2` available.